content/en/docs/Writing policies/background.md

```diff
@@ -39,4 +43,5 @@ When background scanning is enabled, regardless of whether the policy's `validat
 | `validationFailureAction: enforce` | None         | None              |
 | `validationFailureAction: audit`   | Report       | None              |
 
-Also, policy rules that are written using variables from [Admission Review](/docs/writing-policies/variables/#variables-from-admission-review-request-data) request information (e.g. `{{request.userInfo}}`) cannot be applied to existing resources in the background scanning mode since that information is not available. Hence, these rules must set `background` to `false` to disable background scanning.
+
+Policy rules that are written using variables from [Admission Review](/docs/writing-policies/variables/#variables-from-admission-review-request-data) request information (e.g. `{{request.userInfo}}`) cannot be applied to existing resources in the background scanning mode since that information is not available. Hence, these rules must set `background` to `false` to disable background scanning.
```

Comment:
As an aside, is this check being included in future policy schema validations?

Reply:
yes, it already is....